ReportPortal.Client.Tests/LaunchItem/LaunchItemFixture.cs

```diff
@@ -7,12 +7,20 @@
 using ReportPortal.Client.Requests;
 using System.Threading.Tasks;
 using System.Net.Http;
+using ReportPortal.Client.Tests.Base;
 using Xunit;
 
 namespace ReportPortal.Client.Tests.LaunchItem
 {
-    public class LaunchItemFixture : BaseFixture
+    public class LaunchItemFixture : BaseFixture, IClassFixture<LaunchFixtureBase>
```

Comment:
Do we really need to create new extra launch for tests to verify `Launch` functionality? I guess not all tests requires launch item on the server.